qadence_protocols/measurements/tomography.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+import torch
+from qadence import QuantumModel
+from qadence.blocks.utils import unroll_block_with_scaling
+from torch import Tensor
+
+from qadence_protocols.measurements.utils import iterate_pauli_decomposition
+
+
+def compute_expectation(
+    model: QuantumModel,
+    options: dict,
+    param_values: dict[str, Tensor] = dict(),
+) -> Tensor:
+    """Compute expectation values from the model by sampling via rotated circuits in Z-basis.
+
+    Args:
+        model (QuantumModel): Model to evaluate.
+        options (dict): _description_
+        param_values (dict[str, Tensor], optional): _description_. Defaults to dict().
+
+    Raises:
+        TypeError: If observables are not given as list
+        KeyError: If a 'n_shots' kwarg of type 'int' is missing in options.
+
+    Returns:
+        Tensor: Expectation values
+    """
+
+    observables = model._observable
+    if not isinstance(observables, list):
```

Comment:
These two should be tested.

Reply:
What two should be tested???

Reply:
Testing these two cases for error handling.

Reply:
Ah ok, then just done!